make_datasets.py

```diff
@@ -112,6 +133,10 @@ def main():
                              required=True, help='Path to save matrix X')
     regr_parser.add_argument('-y', '--filey', '--fileY', type=str,
                              required=True, help='Path to save vector y')
+    regr_parser.add_argument('-xt', '--filextest', '--fileXtest', type=str,
```

Comment:
Short options must be only one character long. Generally, `-xt` means the same thing as `-x -t`. I would suggest something like `-X` instead of `-xt`. Same goes for `-ts` - could this be `-t`? `-T`? `-s`? `-S`?

Reply:
Parser arguments were corrected.